readthedocs/api/v2/views/model_views.py

```diff
@@ -289,9 +290,26 @@ class BuildViewSetBase(UserSelectViewSet):
     )
     def running(self, request, **kwargs):
         project_slug = request.GET.get('project__slug')
+        query = Q(project__slug=project_slug)
+
+        project = get_object_or_404(Project, slug=project_slug)
```

Comment:
Do we actually want to 404 here? Does the client expect this?

Reply:
I suppose we want a 404 if the project does not exist, yes. However, we are not currently handling any exception in the client yet.